packages/@vue/cli/lib/promptModules/cssPreprocessors.js

```diff
@@ -2,7 +2,7 @@ module.exports = cli => {
   cli.injectFeature({
     name: 'CSS Pre-processors',
     value: 'css-preprocessor',
-    description: 'Add support for CSS pre-processors like SASS, Less or Stylus',
+    description: 'Add support for CSS pre-processors like SASS, Less or Stylus (PostCSS is always included)',
```

Comment:
Note: This will only be displayed in the UI.

Reply:
Oh ^^

Reply:
...in the UI we can go back& forth anyway, right? Then maybe we have to solve this another way.

Don't want to make the `name`prop disproportionately long compared to other prompts with this ...

Reply:
`message` key is used by inquirer.js for the prompts.